src/convertMswMatchToPact.ts

```diff
@@ -24,7 +24,7 @@ export const convertMswMatchToPact =  ({
           headers: headers?.excludeHeaders
             ? omit(match.request.headers['_headers'], headers.excludeHeaders)
             : match.request.headers['_headers'],
-          body: match.request.bodyUsed ? match.request.body : undefined
+          body: match.request.body ? match.request.body : undefined
```

Comment:
```suggestion
          body: match.request.body
```
is enough in this case. If `body` can be either string or undefined, so if body is falsy then it must be undefined

![image](https://user-images.githubusercontent.com/11528830/167089187-9140e205-54a8-495b-8b03-d45cc8935a7c.png)